components/HistoricalWithdrawal.tsx

```diff
@@ -0,0 +1,159 @@
+import { VFC } from "react";
+import { IntrinsicElements } from "@/types";
+import { css } from "@emotion/react";
+import {
+	Theme,
+	Accordion,
+	AccordionSummary,
+	AccordionDetails,
+	TextField,
+	Tooltip,
+} from "@mui/material";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+
+interface HistoricalWithdrawalProps {
+	historicalEventProofId: number;
+	setHistoricalEventProofId: Function;
+	historicalBlockHash: string;
+	setHistoricalBlockHash: Function;
+}
+
+const HistoricalWithdrawal: VFC<
+	IntrinsicElements["div"] & HistoricalWithdrawalProps
+> = (props) => {
+	return (
+		<div {...props} css={styles.root}>
+			<Accordion css={[styles.formSettings]}>
+				<AccordionSummary
+					css={styles.formSettingsSummary}
+					expandIcon={<ExpandMore />}
+				>
+					Advanced
+				</AccordionSummary>
+				<AccordionDetails>
+					<div css={styles.formField}>
+						<label htmlFor="historicalEventProofId">
+							Historical Event Proof Id
+						</label>
+						<TextField
+							css={styles.input}
+							value={props.historicalEventProofId}
+							onChange={(event) =>
+								props.setHistoricalEventProofId(event.target.value)
+							}
+							type="number"
+							InputProps={{
+								endAdornment: <EventProofToolTip field="id" />,
+							}}
+						/>
+						<label htmlFor="historicalBlockHash">Block Hash</label>
+						<TextField
+							css={styles.input}
```

Comment:
Block Hash is relatively long so this input should in full-width, plus add `multiline={true}` prop so it can increase to 2 lines if needs to